backend/controllers/messageControllers.js

```
@@ -48,4 +48,43 @@ const sendMessage = asyncHandler(async (req, res) => {
   }
 });
 
-module.exports = { allMessages, sendMessage };
+const editMessage = asyncHandler(async (req, res) => {
+  const { messageId, newContent } = req.body;
+
+  if (!messageId || !newContent) {
+    console.log("Invalid data passed into request");
```

can you return this error in json?

---

> can you return this error in json?

sure.

---

done